src/styles/base/_03-mixins.scss

```diff
@@ -38,15 +38,6 @@
     max-width: #{$container-max-width};
     padding: #{$v-padding} #{$h-padding};
     width: 100%;
-
-    @include breakpoint(max-width $container-max-width) {
```

Comment:
Looking at this same wrapper mixin in the D8 repo, I think this will work by removing `max-width` from the value being passed into the breakpoint function.

`@include breakpoint($container-max-width)` instead of...
`@include breakpoint(max-width $container-max-width)`

That way this only kicks in _after_ the width reaches the `$xl` break.

Reply:
Unfortunately this will mess with the breakout mixin—looks like it will apply padding at all the lower sizes, whereas we want `content-top` to have no spacing for things like various `Hero` variants.

Reply:
The opposite would be true, no?

In D8, the following applies the padding calculation at (min-width: 1152px):
`@include breakpoint($container-max-width)`

![d8-wrapper](https://user-images.githubusercontent.com/1325185/92492374-46618600-f1c1-11ea-9391-0b0753a76902.png)

In the DS, the wrapper mixin originally had the following:
`@include breakpoint(max-width $container-max-width)`

That would reverse the media query logic...applying the padding calculation only to smaller size screens.

It's possible that padding calculation is not needed, and removing it completely is fine (as done in the PR). But should we then remove it from the mixins in D8 also?

Reply:
> The opposite would be true, no?

I've had some bad merges working on this ticket, so it could be that I'm the one turned around here. I was seeing a padding getting applied on items that had the `breakout` mixin on smaller sizes, which was messing with the point of that mixin.
 
> It's possible that padding calculation is not needed, and removing it completely is fine (as done in the PR). But should we then remove it from the mixins in D8 also?

I would like to keep these two things in parity if possible, but one thing of note is that d8 is using `1152px` as that max breakpoint and we're using `1280px`. I would feel more confident if I understood why that media query was adding padding in the first place...

Reply:
Agreed. I also am not sure what is the point of kicking in this padding at the largest breakpoint.